adapter/src/test/java/com/twilio/kudu/sql/parser/KuduDDLIT.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.kudu.client.KuduException;
 import org.apache.kudu.client.KuduTable;
 import org.apache.kudu.client.PartialRow;
+import org.apache.kudu.client.Partition;
```

Comment:
Looks like an unused import? Spotless only checks the `main` directory for unused imports

Reply:
removed the unused import